dev/DynamicDependency/API/MddDetourPackageGraph.cpp

```diff
@@ -109,6 +125,12 @@ VERSIONHELPERAPI IsWindowsVersionOrGreaterEx(
     return VerifyVersionInfoW(&osvi, VER_MAJORVERSION | VER_MINORVERSION | VER_SERVICEPACKMAJOR | VER_BUILDNUMBER, c_conditionMask) != FALSE;
 }
 
+VERSIONHELPERAPI IsWindows10_19H1OrGreater()
```

Comment:
Shouldn't the code look for a feature and not a version?

Reply:
There isn't a feature check corresponding to this API. It was introduced in 19H1, and the docs say it as such.

Reply:
I think the suggestion is to GetProcAddress poke around and delete all the version checks.

Reply:
Yes, but the lineage of some of this isn't as clearcut as it would appear. I'll make a future PR replacing the version checks if/where possible.